app/uk/gov/hmrc/traderservices/views/StartView.scala.html

```diff
@@ -30,5 +27,49 @@
 
 @govukLayout(pageTitle = Some(messages("view.start.title"))) {
 
+    @html.h1(messages("view.start.heading"))
+
+    @html.h2(messages("view.start.route-one"))
+
+    @html.ul(
+        messages("view.start.route-one.bullets.1"),
+        messages("view.start.route-one.bullets.2"),
+        messages("view.start.route-one.bullets.3")
+    )
+
+    @html.a("/trader-services/route-one", messages("view.start.route-one.link.text"))
+
+    <br><br>
```

Comment:
&lt;br&gt;s shouldn't be used for spacing.

Reply:
I will leave to you as a frontend developer to propose something instead or remove. Feel free to amend the PR.